src/ir/possible-contents.cpp

```diff
@@ -1275,6 +1275,31 @@ struct InfoCollector
   void visitContNew(ContNew* curr) {
     // TODO: optimize when possible
     addRoot(curr);
+
+    // The function reference that is passed in here will be called, just as if
+    // we were a call_ref, except at a potentially later time.
```

Comment:
Alternatively we could collect funcref contents used to allocate each continuation type (using a heap type location?), then only mark them as called when we see a `resume` on their continuation type.

Reply:
Yes, this could definitely be improved. The TODOs are for that. What makes this kind of annoying to do is that cont.bind changes the types... so that will need some care.